examples/src/main/scala/mhtml/examples/Chosen.scala

```diff
@@ -105,10 +105,15 @@ object Chosen {
       div -> candidates
     }
     val rxCandidates: Rx[Seq[T]] = rxCandidatesWithApp.map(_._2)
-    val highlightedCandidate: Rx[T] =
+    val highlightedCandidate: Rx[T] = {
+      val filtered = Var[T](rxCandidates.get.head)
+
       (for { index <- rxIndex; candidates <- rxCandidates } yield {
         candidates.zipWithIndex.find(_._2 == index).map(_._1)
-      }).collect { case Some(x) => x }
+      }).foreach(_.foreach(filtered.:=))
```

Comment:
This looks a bit weird but I'm sure I would have written it differently if `collect` hadn't existed.